### PR TITLE
Completed examples for string manipulation

### PR DESCRIPTION
**Pull request by Evan-Gilbert-1212**
Completed examples for existing missing string methods. Please check and provide any feedback. Once this is approved, I can add any others that may be helpful.

### CODE REVIEW

File: lessons/misc-quick-reference/cs-string.md [hunk 1]
```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
...
----
+We can get the length of a string by using the length property.
+
+```C#
+var name = "Suncoast Developers Guild";
+
+var nameLength = name.Length;
+
+Console.WriteLine(nameLength);
+//Output: 25;
```
Comment (gstark) [inline lessons/misc-quick-reference/cs-string.md:62]: I'd prefer that all comments have at least a single space.

```
// Output: 25
```

And there wouldn't be a `;` on that output...

File: lessons/misc-quick-reference/cs-string.md [hunk 2]
```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
...
+//r
+//y
+```
+
+Alternatively, we can use the string length to use a for loop and access each character by its index.
+
+```C#
+var name = "Mary";
+
+for (var i = 0; i < name.Length; i++)
```
Comment (gstark) [inline lessons/misc-quick-reference/cs-string.md:89]: No single letter variables allowed. So use `index`

File: lessons/misc-quick-reference/cs-string.md [hunk 3]
```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
...
+//M
+//a
+//r
+//y
+```
 
 ### Splitting a string
 
----
+We can split a string into an array of strings using the Split method. The first parameter will be the character you want to split the string on.
```
Comment (gstark) [inline lessons/misc-quick-reference/cs-string.md:103]: Realistically your example is splitting on a string, given you wrote `(" ")` -- double check your usage and if it is on a string, change this. If it should be a character, change the code below.

File: lessons/misc-quick-reference/cs-string.md [hunk 4]
```diff
@@ -49,50 +49,189 @@ var message = $"{name} is a {description}";
...
+
+```C#
+var sentence = "Today is a beautiful day";
+
+var words = sentence.Split(" ");
+
+//words = ['Today', 'is', 'a', 'beautiful', 'day'];
```
Comment (gstark) [inline lessons/misc-quick-reference/cs-string.md:110]: We need some comment here like "The effect would be that `words` is equal to ...."

File: lessons/misc-quick-reference/cs-string.md (hunk 1 above)
Comment (Evan-Gilbert-1212) [reply lessons/misc-quick-reference/cs-string.md]: I corrected this example and added single spaces to all comments throughout my additions.

File: lessons/misc-quick-reference/cs-string.md (hunk 2 above)
Comment (Evan-Gilbert-1212) [reply lessons/misc-quick-reference/cs-string.md]: Still a bad habit I am trying to break! Corrected.

File: lessons/misc-quick-reference/cs-string.md (hunk 3 above)
Comment (Evan-Gilbert-1212) [reply lessons/misc-quick-reference/cs-string.md:103]: Yep, you are correct. Split works on a single character from my experience. I have corrected the example to have single ticks.

File: lessons/misc-quick-reference/cs-string.md (hunk 4 above)
Comment (Evan-Gilbert-1212) [reply lessons/misc-quick-reference/cs-string.md]: I have added "The effect would be:" to all relevant examples I have added.

**Pull request merged**